lib/Util/Security.php

```diff
@@ -13,9 +13,8 @@ static function generateToken($action = 'mailpoet_token') {
   }
 
   static function generateRandomString($length = 5) {
-    // non-cryptographically strong random generator
     return substr(
-      md5(uniqid((string)mt_rand(), true)),
+      bin2hex(random_bytes(16)), // phpcs:ignore
       0,
       min(max(5, (int)$length), 32)
```

Comment:
I think this was here before because we used `md5` and we couldn't generate a string longer than 32. But now using `random_bytes` we can generate anything. So we could maybe fix this line as well. A couple of lines below we call this function with 64. We couldn't generate such string before but now we can.

Reply:
Right. I will fix that.